### ISSUE TITLE
Remove deprecated `null_data_source` datas

### ISSUE THREAD

**Issue by AgustinRamiroDiaz** [terraform-google-modules/terraform-google-event-function#182]
### TL;DR

This project makes use of the `null_data_source` datas, which is deprecated in favor of `locals` ([docs](https://registry.terraform.io/providers/hashicorp/null/latest/docs/data-sources/data_source)). This trigger warnings

### Expected behavior

No warnings are triggered when using this module

### Observed behavior

Warnings are triggered when using this module
![image](https://github.com/terraform-google-modules/terraform-google-event-function/assets/49416765/722e76af-4b2c-4998-9cc8-d5ab3346027c)

### Terraform Configuration

```hcl
Any configuration should trigger the warnings
```

### Terraform Version

```sh
1.5.4
```

### Additional information

I have stumbled upon this while using https://github.com/terraform-google-modules/terraform-google-scheduled-function

This is not a bug per se, but I think it's the most appropriate template

Here are the 2 deprecated `null_data_source`:
- https://github.com/terraform-google-modules/terraform-google-event-function/blob/8d977a4013a09bc35c6ce043f4989fbb3fa6f3b1/examples/automatic-labelling-from-repository/main.tf#L42
- https://github.com/terraform-google-modules/terraform-google-event-function/blob/8d977a4013a09bc35c6ce043f4989fbb3fa6f3b1/main.tf#L42

**Comment by AgustinRamiroDiaz** [comment]
anything I can do to helm get this merged?
